src/main/resources/wa-task-initiation-employment-et_scotland.dmn

```diff
@@ -2179,6 +2179,329 @@ else null</text>
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0dykh8b">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_08tv4ss">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1j54di6">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0n6nmi2">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0113t4c">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0mjsmmj">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_19foadm">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_17i7eqj">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0q01haf">
+          <text>"Change my personal details"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1bh6vrl">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0evmvi3">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0sifvyg">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1odp6rc">
+          <text>"AmendClaimantDetails"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1c6pgf8">
+          <text>"Amend Party Details"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0d6ndzw">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0x0szpo">
+          <text>"Amendments"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1xlqqqk">
+        <inputEntry id="UnaryTests_1u4thlc">
+          <text>"tseClaimantRepResponse"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_03ves0p">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qpto6o">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1uc6zb2">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_16eheb0">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1x62mkr">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1b7n1u1">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1wu5nc3">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1cedwih">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1ol8mt5">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1rctlgn">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_090ahd9">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0i9hrud">
+          <text>"Claimant"</text>
```

Comment:
Noticed this when reviewing callbacks - the response from is set to Claimant Representative and not Claimant so this task might not trigger - https://github.com/hmcts/et-ccd-callbacks/pull/2259/files#diff-f007255afc0d633a2c2099ce178ce90cd4e16e7f2684a76a8d24177611e13700R211

Reply:
addressed comments [e95be0d](https://github.com/hmcts/et-wa-task-configuration/pull/263/commits/e95be0d9a145b0fd0fe51e0b1a9ff52b125285c6)